File: art2_projects/org.kermeta.art2.runtime/src/main/java/org/kermeta/art2/runtime/proxy/Factory.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.kermeta.art2.runtime.proxy;

/**
 *
 * @author ffouquet
 */
public class Factory {
  public static ActiveMQConsumer createConsumer() { return new ActiveMQConsumer(); }

  public static ActiveMQProducerActor createProducer() { return new ActiveMQProducerActor(); }
}

File: art2_projects/org.kermeta.art2.core.bus.activemq/src/main/scala/org/kermeta/art2/runtime/proxy/Factory.scala
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.kermeta.art2.runtime.proxy

object Factory {
  def createConsumer : ActiveMQConsumer = {  new ActiveMQConsumer(); }
  def createProducer : ActiveMQProducerActor = {  new ActiveMQProducerActor(); }
}
